Formula/tea.rb

```diff
@@ -0,0 +1,23 @@
+class Tea < Formula
+  desc "Command-line tool to interact with Gitea servers"
+  homepage "https://gitea.com/gitea/tea"
+  url "https://gitea.com/gitea/tea/archive/v0.8.0.tar.gz"
+  sha256 "6c73c0a7b66cdfd1e5a302257d54df06a3a41eb9bdbfeb547966db431ae23b23"
+  license "MIT"
+  head "https://gitea.com/gitea/tea.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # Remove `if build.head?` on next release.
+    system "make", "vendor" if build.head?
+    system "make"
+    bin.install "tea"
```

Comment:
Is there a `make install` we can use for this?

Reply:
There is, but it seems to use `GOPATH` to set the path into which the binary is installed, and I assumed I shouldn't be changing `GOPATH`: https://gitea.com/gitea/tea/src/branch/master/Makefile#L104

Reply:
I've switched to using `go build` as suggested by @chenrui333.